src/zope/interface/verify.py

```diff
@@ -30,21 +35,21 @@
 MethodTypes = (MethodType, )
 
 
-def _verify(iface, candidate, tentative=0, vtype=None):
-    """Verify that 'candidate' might correctly implements 'iface'.
+def _verify(iface, candidate, tentative=False, vtype=None):
+    """Verify that *candidate* might correctly implement *iface*.
 
     This involves:
 
-      o Making sure the candidate defines all the necessary methods
+      - Making sure the candidate defines all the necessary methods
 
-      o Making sure the methods have the correct signature
+      - Making sure the methods have the correct signature
 
-      o Making sure the candidate asserts that it implements the interface
+      - Making sure the candidate asserts that it implements the interface
```

Comment:
Nitpick: in ReST this translates to `<blockquote><ul>`; if you want to drop the unnecessary `<blockquote`>, you need to unindent the list markers.

(I personally hate that feature of ReST, but I've learned to accept it.)

Reply:
Thanks. I'll tweak it again when I'm working on #171.